js/deer-render.js

```diff
@@ -204,6 +204,255 @@ DEER.TEMPLATES.shadow = (obj, options = {}) => {
+                        if(elem.querySelector(".statusDrawer").classList.contains("is-hidden")){
+                            elem.querySelector(".statusDrawer").classList.remove("is-hidden")
+                        }
+                        else{
+                            elem.querySelector(".statusDrawer").classList.add("is-hidden")
+                        }
+                    })
+
+                    elem.querySelector(".notifyReviewerBtn").addEventListener("click", e => {
+                        if(!userHasRole(["dunbar_user_curator","dunbar_user_contributor","dunbar_user_reviewer"])) { return alert(`This function is limited to contributors, reviewers, and curators.`)}
+                        let proceed = confirm("This action is connected with you username.  Click OK to proceed and add your note.")
+                        if(!proceed){return}
+                        let url = DEER.URLS.CREATE
+                        let method = "POST"
+                        const commentAnno = {
+                            "@context": "http://purl.org/dc/terms",
+                            "motivation" : "commenting",
+                            "type": "Annotation",
+                            "body":{
+                                "comment":{
+                                    "type" : "Comment",
+                                    "text" : elem.querySelector("textarea").value,
+                                    "author" : DLA_USER["http://store.rerum.io/agent"]
+                                }
+                            },
+                            "target":obj["@id"]
+                        }
+                        if(obj?.comment.source.citationSource){
+                            url = DEER.URLS.UPDATE
+                            method = "PUT"
+                            commentAnno["@id"] = obj.comment.source.citationSource
+                        }
+                        fetch(url, {
+                            method: method,
+                            headers: {
+                                "Content-Type": "application/json",
+                                "Authorization": `Bearer ${DLA_USER.authorization}`
+                            },
+                            body: JSON.stringify(commentAnno)
+                        })
+                        .then(response => response.json())
+                        .then(anno => {
+                            statusComment.querySelector("span").innerHTML = "See your status comment"
+                            statusComment.querySelector("pre").innerHTML = elem.querySelector("textarea").value
+                            addRecordToManagedList(obj, elem, coll)
+                        })
+                        .catch(err => { })
+                        
+                    })
+
+                    elem.querySelector(".statusCommentText").addEventListener("keydown", e => {
+                        if( e.keyCode === 27 ) {
```

Comment:
Are you using AI? Why does <kbd>ESC</kbd> cancel the comment and submit anyway? Keyboard shortcuts aren't really in scope right now...